src/main/java/net/mcreator/ui/modgui/EnchantmentGUI.java

```diff
@@ -59,6 +59,8 @@
 	private final JCheckBox isTreasureEnchantment = L10N.checkbox("elementgui.common.enable");
 	private final JCheckBox isCurse = L10N.checkbox("elementgui.common.enable");
 	private final JCheckBox isAllowedOnBooks = L10N.checkbox("elementgui.common.enable");
+	private final JCheckBox canGenerateInLootTables = L10N.checkbox("elementgui.common.enable");

```

Comment:
This should be true by default as so far it was true, same with other param. i think (check the code). Make sure to set the default value in Enchantment.java too so old workspaces are migrated properly too

Reply:
I don't know what to do in Enchantment.java to migrate old workspaces...

Reply:
In constructor (without argument), set fields to true. Check some otherelements how they do it